media-libs/libggiwmh/libggiwmh-0.3.2-r2.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Window manager hints extensions for libggi"
+HOMEPAGE="https://ibiblio.org/ggicore/packages/libggiwmh.html"
+SRC_URI="https://downloads.sourceforge.net/ggi/${P}.src.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="X"
+
+RDEPEND=">=media-libs/libggi-2.2.2
+	X? (
```

Comment:
This likely should be USE=gui, see https://projects.gentoo.org/qa/policy-guide/use-flags.html#pg0802.